src/main/java/com/kainos/ea/dao/JobsDao.java

```diff
@@ -12,8 +12,8 @@
 public class JobsDao {
     public List<Job> getJobs(Connection c) throws SQLException, DatabaseConnectionException {
         Statement st = c.createStatement();
-        PreparedStatement statement = c.prepareStatement("SELECT id, job_roles.name, description, specification, responsibilities, bands.name, capability " +
-                "FROM job_roles JOIN bands USING(id)");
+        PreparedStatement statement = c.prepareStatement("SELECT job_roles.id, job_roles.name, description, specification, responsibilities, bands.name, capability " +
+                "FROM job_roles JOIN bands ON job_roles.band_id = bands.id");
```

Comment:
Did we agree to bring this in on this ticket?

Reply:
Ah yes I forgot it was being moved to another adhoc ticket I'll revert it here now